CoinDock-app/src/Screens/Account/settings/AccountSettings.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import "../Account.css";
+import { FaArrowLeft} from 'react-icons/fa';
+import { FaEdit } from 'react-icons/fa';
+import "../../../Shared/common-styles/button.css"
+import { useAccount } from "App/Api/accapi";
+import { useNavigate } from "react-router-dom";
+import { Card } from "react-bootstrap";
+function AccountSettings (){
+  const { data: account}= useAccount();
+  const accountDetails = account?.data?.results?.user || {};
+    const navigate = useNavigate();
+    const fields= [
+      {
+        label:'Email',
+        fieldKey:'email',
+       },
+       {
+        label:'Change Password',
+        fieldKey:'changePassword',
+       },
+       {
+        label:'Recovery Code',
+        fieldKey:'recoverycode',
+       },
+
+    ]
+    const handlePassword =() =>{
+        navigate("/apassword")
+    }
+
+    return (
+      <>
+       <div type="submit" className="cd-back"onClick={()=>{navigate("/account")}}>< FaArrowLeft/></div>
+      <h2 className="cd-headerStyle">Account settings</h2>
+        {fields.map((field,id)=> (
+            <div className="cd-card1" key={id}>
+              {field.fieldKey === 'email' ?
```

Comment:
Here, also same as accounts.js 
You can add display display label in the array of objects and pass it to.
And can you please format it with proper indentations and spacing please.

Reply:
Modified